api/controllers/fda_applications.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const FdaApplication = require('../models/fda_application');
+
+function getFdaApplication(req, res) {
+  const id = req.swagger.params.id.value;
+
+  return new FdaApplication({ id: id }).fetch({
+      withRelated: ['fda_approvals', 'organisation'],
+    })
+    .catch((err) => {
+      res.finish();
+      throw err;
+    })
+    .then((fda_application) => {
+      if (fda_application) {
+        res.json(fda_application);
+      } else {
+        res.status(404);
+        res.finish();
+      }
+    });
+}
+
+function getFdaApplications(req, res) {
```

Comment:
To keep naming consistenty with https://github.com/opentrials/api/blob/master/api/controllers/sources.js, name it `listFdaApplications`